frontend/src/components/counter/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useTimer } from "react-timer-hook";
+import boxOffice from "@assets/boxOffice.png";
+import logoSah from "@assets/Logo_SAH.png";
+import SMyTimer from "./style";
+
+function MyTimer({ expiryTimestamp }) {
+  const { minutes, hours, days } = useTimer({
+    expiryTimestamp,
+    onExpire: () => console.warn("onExpire called"),
+  });
+
+  return (
+    <SMyTimer>
+      <div>
+        <h3>PROCHAIN MATCH</h3>
+      </div>
+
+      <ul className="nextMatch">
+        <img src={logoSah} alt="équipe à l'extérieur" />
+        <li>VS</li>
+        <img src={logoSah} alt="équipe à l'extérieur" />
+      </ul>
+      <ul className="dayTime">
+        <li>J1 </li>
+        <p>SAMEDI 3 SEPTEMBRE</p>
+        <p>19H30</p>
+        <p>SALLE MAURICE HUGOT</p>
+      </ul>
+      <div className="timer">
+        <ul>
+          <li className="numbersCount"> {days} </li>
+          <li>JOURS</li>
+        </ul>
+        <span>-</span>
+        <ul>
+          <li className="numbersCount"> {hours} </li>
+          <li>HEURES</li>
+        </ul>
+        <span>-</span>
+        <ul>
+          <li className="numbersCount"> {minutes} </li>
+          <li>MINUTES</li>
+        </ul>
+      </div>
+      <div className="billetDiv">
+        <a href="https://www.sah-ph.fr/billetterie/">
+          <img className="boxOffice" src={boxOffice} alt="billeterie" />
+        </a>
+      </div>
+    </SMyTimer>
+  );
+}
+
+export default function App() {
```

Comment:
Wtf ? Why do you export a second app ? o.O

Reply:
same